src/handlers/shared/check-task-stale.ts

```diff
@@ -1,9 +1,11 @@
-export function checkTaskStale(staleTask: number, createdAt: string) {
-  if (staleTask !== 0) {
-    const days = Math.floor((new Date().getTime() - new Date(createdAt).getTime()) / (1000 * 60 * 60 * 24));
-    const staleToDays = Math.floor(staleTask / (1000 * 60 * 60 * 24));
-    return days >= staleToDays && staleToDays > 0;
+export function checkTaskStale(staleTaskMilliseconds: number, createdAt: string): boolean {
+  if (staleTaskMilliseconds === 0) {
+    return false;
   }
 
-  return false;
+  const currentDate = new Date();
+  const createdDate = new Date(createdAt);
+  const millisecondsSinceCreation = currentDate.getTime() - createdDate.getTime();
+
+  return millisecondsSinceCreation >= staleTaskMilliseconds && staleTaskMilliseconds > 0;
```

Comment:
Maybe this check is redundant since you return `false` when `staleTaskMilliseconds` is equal to zero